.github/workflows/platformDeploy.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     if: ${{ fromJSON(needs.validateActor.outputs.IS_DEPLOYER) }}
     runs-on: ubuntu-latest
     steps:
-      - uses: Expensify/App/.github/actions/composite/setupNode@main
+      - uses: Expensify/App/.github/actions/composite/setupNode@andrew-npm-8
```

Comment:
@roryabraham - I was unsure what to do here, if we leave as `main`, it will use an older version of `node` and fail. So I think this is necessary until we merge to `main`, then change it back.

Reply:
Another alternative would be to just checkout and use the local version of the action (no @ tag). But honestly I think this option is fine – we can just do a fast-follower PR to switch it back to main

Reply:
Sounds good 👍